workflow/rocoto/gfs_tasks.py

```diff
@@ -934,8 +883,8 @@ def _fcst_cycled(self):
 
         dependencies = rocoto.create_dependency(dep_condition='and', dep=dependencies)
 
-        if self.run in ['gdas']:
-            dep_dict = {'type': 'cycleexist', 'condition': 'not', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
+        if self.run in ['gdas', 'enkfgdas']:
```

Comment:
Why is `enkfgdas` a dependency here? This creates either the `gdasfcst` or the `gfsfcst` task for the cycled.

Reply:
Also, `gdasstage_ic` will only exist for the first half cycle, so the offset to check if the previous cycle exists should stay, no?

Reply:
> Why is enkfgdas a dependency here? This creates either the gdasfcst or the gfsfcst task for the cycled.

Removed @ a53c97e2, thanks!

> Also, gdasstage_ic will only exist for the first half cycle, so the offset to check if the previous cycle exists should stay, no?

The check for a prior cycle is only needed for the fcst job when there aren't any jobs preceding it. Now that the staging job runs before it, we don't need the prior cycle check. The check of the staging job completion replaces it.

Should we make it possible to disable the staging job, then yes, we'd want the prior cycle check for the first half cycle fcst job again.